.github/workflows/runexamples.yaml

```diff
@@ -12,9 +12,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v1
+      - uses: actions/setup-python@v2
         with:
-          version: '3.x'
+          python-version: 3.8.6
```

Comment:
I would not pin version 3.8 explicitly here.

Reply:
Ok. I see you did that to avoid 3.9. Could you do "3.8" instead ... just to be the least specific necessary.